tests/compiler_test.go

```diff
@@ -0,0 +1,17 @@
+// +build js
+
+package tests
+
+import (
+	"testing"
+)
+
+func TestVariadicNil(t *testing.T) {
+	printVari := func(strs ...string) []string {
+		return strs
+	}
+
+	if v := printVari(); v != nil {
```

Comment:
It'd be nice to be more consistent and stick with the `got, want` test style:

```Go
if got, want := printVari(), []string(nil); got != want {
	t.Errorf("printVari() returned: %v, want: %v", got, want)
}
```

Reply:
I'm intentionally using untyped `nil` here; but I've renamed the `got` variable.